src/pages/index.mdx

```diff
@@ -1,14 +1,18 @@
+import Image from 'next/image';
+import logo from '../images/teams_logo.svg';
+import teamsScreenshot from '../images/teams_app_screenshot.png';
+
```

Comment:
@andresaraujo I _think_ we can also do this with `<MDXProvider>` from `@mdx-js/react` by defining the components.

```
import { MDXProvider } from '@mdx-js/react'
import Image from 'next/image'

const components = {
  img: Image,
  // We could define other customizations here if we need to
}

<MDXProvider components={components}>
```

This would mean we do not have to do any imports and use the components at all. It MIGHT also mean the assets need to go back to the public dir.

Did we evaluate this approach instead of importing the component to the MDX?

Reply:
@brianespinosa Just updated the PR with this approach. Thanks.

Now you can either use the md image syntax
`![Teams Logo](images/teams_logo.svg "Teams Logo")`

Or the html image syntax
<img src="/images/teams_logo.svg" alt="Teams Logo" width={100} height={100} />

Note:
- I added a default minimum image width/height to 200x200. `next/image` requires both. So in case you use the md syntax where there is no way to forward the props I add the minimum size. 
- Also if you use the img syntax and dont specified both width and height I complement the missing attribute. 
- See https://github.com/Espressive/help-teams/pull/3/files#diff-88a36bc7a53bfa58c6f451464798d631d785160dfacfdaf9479be7a0b1d7e5e5R21